Traceback (most recent call last):
  File "C:\Users\VB\Documents\GitHub\TPRO\main.py", line 19, in <module>
    student = Listener(repo)
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\listener.py", line 17, in __init__
    trpo_repo.check_issues()
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\repository.py", line 149, in check_issues
    issue.edit(
  File "C:\Users\VB\AppData\Local\Programs\Python\Python38-32\lib\site-packages\github\Issue.py", line 417, in edit
    headers, data = self._requester.requestJsonAndCheck(
  File "C:\Users\VB\AppData\Local\Programs\Python\Python38-32\lib\site-packages\github\Requester.py", line 317, in requestJsonAndCheck
    return self.__check(
  File "C:\Users\VB\AppData\Local\Programs\Python\Python38-32\lib\site-packages\github\Requester.py", line 342, in __check
    raise self.__createException(status, responseHeaders, output)
github.GithubException.GithubException: 502 {"message": "Server Error"}
